packages/react/menu/src/Menu.tsx

```diff
@@ -296,14 +404,16 @@ const MenuContentImpl = React.forwardRef((props, forwardedRef) => {
                     loop={loop}
                     currentTabStopId={currentItemId}
                     onCurrentTabStopIdChange={setCurrentItemId}
-                    // we override the default behaviour which automatically focuses the first item
-                    onEntryFocus={(event) => event.preventDefault()}
+                    onEntryFocus={(event) => {
+                      if (!focusFirstItem) event.preventDefault();
+                    }}
                   >
                     <PopperPrimitive.Content
                       role="menu"
                       {...focusScopeProps}
                       {...contentProps}
```

Comment:
Oh yeah, while testing i noticed that we don't pass the `dir` prop to the DOM node here which seems like a bug. Would be good to pass that from context while you're here if you wouldn't mind otherwise `dir` on the document behaves differently to `dir` as a prop.